src/main/java/by/itech/lab/supplier/controller/pre_tests.http

```diff
@@ -77,53 +77,77 @@ Cookie: {{test}}
 {"name":"New name"}
 
 ###
+#---------------------------------------------Warehouse----------------------------------------------------------------
 
-#GET http://localhost:8080/customers/3/warehouses
-#Content-Type: application/json
-#Connection: keep-alive
-#Cookie: {{test}}
+POST http://localhost:8080/customers/3/warehouses
+Content-Type: application/json
+Connection: keep-alive
+Cookie: {{test}}
+
+{
+  "customerId": "3",
+  "identifier": "123",
+  "type" : "CCC",
+  "totalCapacity" : "44",
+  "addressDto" : {
+    "city" : "Kanzas",
+    "addressLine1" : "Linkoln Str.",
+    "addressLine2" : "44",
+    "state" : {
+      "id" : "5",
+      "state": "Alaska"
+    }
+  },
+  "usersId" : [2]
+}
 
 ###
 
-#GET http://localhost:8080/customers/3/warehouses/2
-#Content-Type: application/json
-#Connection: keep-alive
-#Cookie: {{test}}
+PUT http://localhost:8080/customers/3/warehouses/1
+Content-Type: application/json
+Connection: keep-alive
+Cookie: {{test}}
+
+{
+  "id" : "1",
+  "type" : "anotherType",
+  "totalCapacity" : "10",
+  "addressDto" : {
+    "city" : "Kanzas",
+    "addressLine1" : "Linkoln Str.",
+    "addressLine2" : "66",
+    "state" : {
+      "id" : "6",
+      "state": "Arizona"
+    }
+  }
+}
 
 ###
 
-#POST http://localhost:8080/customers/3/warehouses
-#Content-Type: application/json
-#Connection: keep-alive
-#Cookie: {{test}}
-#{
-#  "identifier": "someIdentifier",
-#  "type": "someType",
-#  "totalCapacity": "8"
-#}
+GET http://localhost:8080/customers/3/warehouses
+Content-Type: application/json
+Connection: keep-alive
+Cookie: {{test}}
 
 ###
 
-#PUT http://localhost:8080/customers/3/warehouses/8
-#Content-Type: application/json
-#Connection: keep-alive
-#Cookie: {{test}}
-#{
-#  "identifier": "777",
-#  "type": "C",
-#  "totalCapacity": "7"
-#}
+GET http://localhost:8080/customers/3/warehouses/1
+Content-Type: application/json
+Connection: keep-alive
+Cookie: {{test}}
 
 ###
 
-#DELETE http://localhost:8080/customers/3/warehouses/8
-#Content-Type: application/json
-#Connection: keep-alive
-#Cookie: {{test}}
+DELETE http://localhost:8080/customers/3/warehouses/1
+Content-Type: application/json
+Connection: keep-alive
+Cookie: {{test}}
 
 ###
+#----------------------------------------------------------------------------------------------------------------------
```

Comment:
?

Reply:
we use this to logically separate our classes. and its easier to navigate

Reply:
look below, there is a comment inside of this separator